maf/pom.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+    <project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>master</artifactId>
+        <groupId>org.mskcc.cbio</groupId>
+        <version>0-unknown-version-SNAPSHOT</version>
```

Comment:
Is there a reason to name a version like this?

Reply:
good question: we have some maven plugin that automatically updates all the version labels based on the github tag/commits. This is just a place holder. I picked that name at some point, should've prolly used something like `0-this-version-is-automtatically-set-by-plugin-x-SNAPSHOT`, might be good to still change